l10n_jp_account_payment_term_cutoff_date/models/account_payment_term_line.py

```diff
@@ -0,0 +1,21 @@
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+
+class AccountPaymentTermLine(models.Model):
+    _inherit = "account.payment.term.line"
+
+    cutoff_date = fields.Integer(
```

Comment:
`day` would better represent what it is.
```suggestion
    cutoff_day = fields.Integer(
```